src/services/Business/BusinessService.ts

```diff
@@ -0,0 +1,28 @@
+import {Api, SEARCH_BUSINESS} from '../../api';
+import {
+  GetBusinessesResponseError,
+  GetBusinessesResponseSuccess,
+  SearchBusinessesSortBy,
+} from './BusinessService.types';
+
+function getAllBusinesses(
+  location: string,
+  default_error: boolean = true,
```

Comment:
Order of function parameters matter. Does this have to be second or last?

Reply:
I kept it in the second order as sort_by, limit, offset might be constant for maximum api calls and for that reason i kept it in that order.